test/TestInit.c

```diff
@@ -75,7 +75,7 @@ void test_init_success(void)
     // Reset EOC and NewData
     mlx90632_i2c_write_ExpectAndReturn(MLX90632_REG_STATUS, reg_status_mock & ~0x01, 0);
 
-    TEST_ASSERT_EQUAL_INT(0, mlx90632_init());
+    TEST_ASSERT_EQUAL_INT(ERANGE, mlx90632_init());
```

Comment:
I dont agree with this. It should return 0 as before, only when extended range is enabled it should return ERANGE.

Reply:
Yes, it makes sense indeed.